### DISCUSSION TITLE
윈도우에서 GENERATE_SOURCEMAP=false로 설정

### DISCUSSION THREAD

**Discussion by ChrisLee02** [wafflestudio/seminar-2021#557] [Show and tell]
https://stackoverflow.com/questions/57093235/generate-sourcemap-false-issue
오류가 뜨길래 찾아봤더니 강의 자료 코드는 리눅스 기준이길래 공유합니다:)

**Comment by woohm402** [comment]
오 이런 거 좋아요 🧇 🧇

**Comment by woohm402** [comment]
추가로, 앞으로도 `Windows OS`에서 발생하는 이슈 및 예외들에 대해서는 대응하거나 정보를 제공드리지 않을 예정이니 윈도우 사용중이신 분들은 참고 부탁드립니다! 제가 잘 몰라요
